examples/tensorgraph/test_mnist.py

```diff
@@ -1,115 +0,0 @@
-import unittest
```

Comment:
this test wasn't getting run -- we only ran the nosetests inside of deepchem/deechem

if one did try to run this it would just throw errors.